client/webpack.config.node.js

```diff
@@ -163,6 +163,7 @@ const webpackConfig = {
 			BUILD_TIMESTAMP: JSON.stringify( new Date().toISOString() ),
 			COMMIT_SHA: JSON.stringify( commitSha ),
 			'process.env.NODE_ENV': JSON.stringify( bundleEnv ),
+			__i18n_text_domain__: JSON.stringify( 'default' ),
```

Comment:
This seems to have been missing for the node build config. Without this, package components (such as Search) break because they use `__i18n_text_domain__` for localization.